docs/framework/unmanaged-api/wmi/qualifierset-put.md

```diff
@@ -2,47 +2,49 @@
 title: QualifierSet_Put function (Unmanaged API Reference)
 description: The QualifierSet_Put function writes the named qualifier and its value.
 ms.date: "11/06/2017"
-api_name: 
+api_name:
   - "QualifierSet_Put"
-api_location: 
+api_location:
   - "WMINet_Utils.dll"
-api_type: 
+api_type:
   - "DLLExport"
-f1_keywords: 
+f1_keywords:
   - "QualifierSet_Put"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "QualifierSet_Put function [.NET WMI and performance counters]"
-topic_type: 
+topic_type:
   - "Reference"
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
+
 # QualifierSet_Put function
-Writes the named qualifier and value. The new qualifier overwrites the previous value of the same name. If the qualifier does not exist, it is created. 
+
+Writes the named qualifier and value. The new qualifier overwrites the previous value of the same name. If the qualifier does not exist, it is created.
 
 [!INCLUDE[internalonly-unmanaged](../../../../includes/internalonly-unmanaged.md)]
-  
-## Syntax  
-  
-```  
+
+## Syntax
+
+```cpp
 HRESULT QualifierSet_Put (
-   [in] int                  vFunc, 
-   [in] IWbemQualifierSet*   ptr, 
+   [in] int                  vFunc,
+   [in] IWbemQualifierSet*   ptr,
    [in] LPCWSTR              wszName,
    [in] VARIANT*             pVal,
    [in] LONG                 lFlavor
-); 
-```  
+);
+```
 
 ## Parameters
 
-`vFunc`   
+`vFunc`
```

Comment:
The two spaces at the end create a soft return in these cases and removing them causes that to go away. 
![image](https://user-images.githubusercontent.com/12971179/53714479-6dd4d700-3e03-11e9-85c6-cea3dde35ede.png)

Perhaps escape like the following?
```suggestion
`vFunc`\
```